http/http_client.hpp

```diff
@@ -763,9 +782,10 @@ class ConnectionPool : public std::enable_shared_from_this<ConnectionPool>
                 }
                 else
                 {
-                    BMCWEB_LOG_DEBUG << "Reusing existing connection "
-                                     << commonMsg;
-                    conn->doResolve();
+                    BMCWEB_LOG_ERROR << "Reusing existing connection "
+                                     << commonMsg << "with state "
```

Comment:
need spaces when you add the values. Else it becomes unreadable at journal traces

Reply:
Ack